data/src/main/java/com/microsoft/azure/kusto/data/auth/ConnectionStringBuilder.java

```diff
@@ -325,4 +329,4 @@ public static ConnectionStringBuilder createWithAadManagedIdentity(String cluste
         csb.useManagedIdentityAuth = true;
         return csb;
     }
-}
\ No newline at end of file
+}
```

Comment:
What's it doing here? Forcing a new line at the end of the file? If so, I don't know of any practical reasons why we'd want this.

Reply:
New lines at the end of files have always been part of formatting standard.
In C it's actually required by the language (though most compilers will ignore it unless you compile with pedantic).
It let's you `cat` files more easily, and gives you better diffs/blames with git.
And again, basically every code standard does them.

Reply:
Yeah I know about it being required in C and that many projects force it, but I'm asking if there are any practical reasons why we'd want this. Anything that affects us or our users such that forcing this adds value.

Diffing/blames shouldn't be relevant, as in Java it's uncommon to have multiple classes in 1 file, and I don't remember any of us ever adding a new class to the end of a file.
`cat`ting a file without a newline at the end just causes the prompt to remain on the previous line. NDB, IMHO. Plus, whomever among us/our users who develop in Linux would most likely use `more` instead of `cat`.